signature-aggregator/config/config.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package config
```

Comment:
Should this `config` package (or just its files) go under `api`? IIUC, they only apply to the aggregation API itself. That structure may better reflect that.

Reply:
We will have metrics port + cache memory limit here soon. If it was just metrics than I'd say move it but Cache memory limit is not API specific